src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metricsdb/MetricsDB.java

```diff
@@ -239,6 +249,21 @@ public void putMetric(Metric<Double> metric, Dimensions dimensions, long windowS
     return create.select().from(DSL.table(metric)).fetch();
   }
 
+  public Result<Record> queryMetric(String metric, Collection<String> dimensions, int limit) {
+    if (!DBUtils.checkIfTableExists(create, metric)) {
+      return null;
```

Comment:
Can we return empty Object instead of null ?

Reply:
We return a null here to stay consistent with the other queryMetric method:
public Result<Record> queryMetric(List<String> metrics, List<String> aggregations, List<String> dimensions) throws Exception